### PR TITLE
Create a header link to API documentation

### PR DESCRIPTION
**Pull request by Whiteaj36**
# What this is doing
Adding a link to the official discord.js.org API documentation to the header of this guide.

# Why is this needed
Addressing [Issue 239](https://github.com/discordjs/guide/issues/239). When I was first researching how to create a discord bot, I was digging through Discord's official API and google searching for any javascript implementations. If you google how to create a discord bot, you end up being brought into the discordjs.guide site much more often (better SEO?).  The discoverability of the package's API is hard to find from that site and it took me a bit more googling to find the actual documentation. While there are links to the site from different pages in discordjs.guide, a top level link for more developers that are familiar with working with APIs would be helpful.

tl;dr; unless you know what to google, discord.js.org is a harder site to find than discordjs.guide, so I want to add a link in the header to find the official docs.

# Screenshot
Screen Shot 2019-07-01 at 5 23 10 PM

### CODE REVIEW

File: guide/.vuepress/config.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ const config = {
 				text: 'Commando',
 				link: '/commando/',
 			},
+			{
+				text: 'API Documentation',
```
Comment (Whiteaj36) [inline guide/.vuepress/config.js]:
```suggestion
				text: 'Discord.js Documentation',
```
